e2e/custom.ini

```diff
@@ -0,0 +1,10 @@
+app_mode = development
+
+[paths]
+
+allow_loading_unsigned_plugins = true
+plugins = /data/plugins
+
+# For "console" mode only
+[log.console]
+level = debug
```

Comment:
Do you need an extra whitespace line here?

Reply:
I actually don't need this line at all - it was for debugging. Good catch!